Program.cs

```diff
@@ -1,19 +1,43 @@
 ﻿// ---------------------------------------------------------------------------------------
 // Training ~ A training program for new joinees at Metamation, Batch- July 2023.
 // Copyright (c) Metamation India.                                                
-// ---------------------------------------------------------------------------
+// ---------------------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to find the number of chocolates that can be bought with the money available and the wrappers available.
+// Money available, price of a chocolate and required wrappers for a chocolate exchange are taken as inputs.  
+// The maximum number of chocolates you can get along with any unused money and wrappers is displayed. 
 // ---------------------------------------------------------------------------------------
-using static System.Console;
+
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Chocolate Wrappers</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method gets required input from the user and displays the number of chocolates, unused money and wrappers as output</summary>
+      static void Main () {
+         Console.WriteLine ("Enter the (M)oney available, (P)rice of chocolate, (W)rappers required to exchange a chocolate in the format (M P W): ");

```

Comment:
Have a separate function to accept an integer input. This function can handle all the checks to ensure the number input is valid.